src/services/imagesRequest.ts

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+export const getImages = async (page: number) => {
+  const res = await axios.get(
+    `https://picsum.photos/v2/list?page=${page}&limit=10`,
```

Comment:
please check how we can pass params to get request with axios, also we can define a limit and you func not provide this feature ...